devel/ansible/roles/hotness-dev/tasks/main.yml

```diff
@@ -1,39 +1,47 @@
 ---
 
 - name: Install helpful development packages
-  dnf: name={{ item }} state=present
-  with_items:
-    - git
-    - ngrep
-    - nmap-ncat
-    - python3-rpdb
-    - tmux
-    - tree
-    - vim-enhanced
-    - tox
-    - rpm-devel
-    - gcc
-    - krb5-devel
+  dnf:
+    name: [
+      git,
+      ngrep,
+      nmap-ncat,
+      python3-rpdb,
+      tmux,
+      tree,
+      vim-enhanced,
+      tox,
+      rpm-devel,
+      gcc,
+      krb5-devel,
+      python3-black
+    ]
+    state: present
 
 - name: Install the-new-hotness development packages
-  dnf: name={{ item }} state=present
-  with_items:
-    - fedmsg
-    - fedmsg-hub
-    - fedmsg-relay
-    - fedpkg
-    - koji
-    - rpm-python
-    - python3-bugzilla
-    - python3-dogpile-cache
-    - python3-flake8
-    - python3-mock
-    - python3-requests
-    - python3-sh
-    - python3-six
-    - python-fedmsg-meta-fedora-infrastructure
-    - python3-fedmsg
-    - rpmdevtools
+  dnf:
+    name: [
+      fedmsg,
+      fedmsg-hub,
+      fedmsg-relay,
```

Comment:
Do you still need those 3?

Reply:
It's question if we want to remove the fedmsg in one go or leave the fedmsg for while.